app/javascript/dashboard/routes/dashboard/conversation/ConversationInfo.vue

```diff
@@ -74,37 +74,32 @@ const staticElements = computed(() =>
 
 <template>
   <div class="conversation--details">
-    <ContactDetailsItem
```

Comment:
Why the need to move this down?

Reply:
@scmmishra I added the conversation information component as a slot inside the custom attributes component because when I moved the [styles](https://github.com/chatwoot/chatwoot/pull/9919/files#diff-2320f582890a2f838308e3f1b6823e5307272ecfe9c25c2132fe99a88aaacc71L191) to inline, I noticed an issue with the odd/even background color in the conversation info section, as shown in the screenshot. This issue is specific to the Conversation Information accordion, which includes both the static conversation info element and the custom attributes section, visible in the screenshot.

**Screenshot**
![image](https://github.com/user-attachments/assets/b9e0050e-369d-4ae8-a004-23730165ca91)

Reply:
There should be a better way to do this, perhaps we can discuss on call. Selecting the background color of each item should ideally be done by the parent in this case with a class like this

`[&>*:nth-child(odd)]:bg-gray-200 [&>*:nth-child(even)]:bg-white`

Reply:
@scmmishra Done